### PR TITLE
docs(extends-user-docs): no-code-change

### PR DESCRIPTION
**Pull request by m4sterbunny**
na

issue-894

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,182 +1,66 @@
...
-```shell
-cargo install --git ssh://git@github.com/ConsenSys/corset --locked --force
-```
+Linea's stack is made up of multiple repositories, these include:
 
-### Update Constraints [Submodule](https://github.com/Consensys/zkevm-constraints/)
+- [linea-monorepo](https://github.com/Consensys/linea-monorepo): The main repository for the Linea stack & network 
+- [linea-besu](https://github.com/Consensys/linea-besu): Fork of Besu to implement the Linea-Besu client
+- [linea-sequencer](https://github.com/Consensys/linea-sequencer): A set of Linea-Besu plugins for the sequencer and RPC nodes
+- [linea-arithmetization](https://github.com/Consensys/linea-arithmetization): Linea-Besu plugin which produces the traces to which the constraint system applies and that serve as inputs to the prover
```
Comment (m4sterbunny) [inline README.md]:
```suggestion
- [linea-tracer](https://github.com/Consensys/linea-tracer): Linea-Besu plugin which produces the traces that the constraint system applies and that serve as inputs to the prover
```

File: README.md [hunk 2]
```diff
@@ -1,182 +1,66 @@
...
-```shell
-cargo install --git ssh://git@github.com/ConsenSys/corset --locked --force
-```
+Linea's stack is made up of multiple repositories, these include:
 
-### Update Constraints [Submodule](https://github.com/Consensys/zkevm-constraints/)
+- [linea-monorepo](https://github.com/Consensys/linea-monorepo): The main repository for the Linea stack & network 
+- [linea-besu](https://github.com/Consensys/linea-besu): Fork of Besu to implement the Linea-Besu client
+- [linea-sequencer](https://github.com/Consensys/linea-sequencer): A set of Linea-Besu plugins for the sequencer and RPC nodes
+- [linea-tracer](https://github.com/Consensys/linea-tracer): Linea-Besu plugin which produces the traces that the constraint system applies and that serve as inputs to the prover
```
Comment (m4sterbunny) [inline README.md]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -1,182 +1,66 @@
...
-```
+Discover [existing plugins](docs/plugins.md) and understand the [plugin release process](docs/plugin-release.md). 
 
-### Install Corset
+## Looking for the Linea code?
 
-```shell
-cargo install --git ssh://git@github.com/ConsenSys/corset --locked --force
-```
+Linea's stack is made up of multiple repositories, these include:
```
Comment (m4sterbunny) [inline README.md]:
```suggestion
- [linea-tracer](https://github.com/Consensys/linea-tracer): 
> Tracing refers to the process of extracting data from the execution of an EVM client in order to construct large matrices known as execution traces. Execution traces are subject to the constraint system specified in the [linea-specification](https://github.com/Consensys/linea-specification) repo and implemented in the [linea-constraints](https://github.com/Consensys/linea-constraints) repo.
>
> This repository contains the elements of the Linea stack responsible for this process.
```

File: README.md [hunk 4]
```diff
@@ -1,182 +1,68 @@
-# Linea Arithmetization (zkEVM)
+# Linea arithmetization (zkEVM)
```
Comment (m4sterbunny) [inline README.md]:
```suggestion
# Linea tracer (zkEVM)
```

File: README.md [hunk 5]
```diff
@@ -1,182 +1,68 @@
...
->
-> - Getters/Setters (via [`@Getter/@Setter`](https://projectlombok.org/features/GetterSetter))
-> - Class log instances (via [`@Slf4j`](https://projectlombok.org/features/log))
-> - Builder classes (via [`@Builder`](https://projectlombok.org/features/Builder))
-> - Constructors (
->   via [`@NoArgsConstructor/@RequiredArgsConstructor/@AllArgsConstructor`](https://projectlombok.org/features/constructor))
-> - etc.
+Linea's stack is made up of multiple repositories, these include:
+- [linea-tracer](https://github.com/Consensys/linea-tracer): 
+> Tracing refers to the process of extracting data from the execution of an EVM client in order to construct large matrices known as execution traces. Execution traces are subject to the constraint system specified in the [linea-specification](https://github.com/Consensys/linea-specification) repo and implemented in the [linea-constraints](https://github.com/Consensys/linea-constraints) repo.
```
Comment (m4sterbunny) [inline README.md]:
```suggestion
```

File: README.md [hunk 6]
```diff
@@ -1,182 +1,67 @@
-# Linea Arithmetization (zkEVM)
+# Linea tracer (zkEVM)
 
-A Linea tracing implementation for [Hyperledger Besu](https://github.com/hyperledger/besu) based on
-an [existing implementation in Go](https://github.com/Consensys/zk-evm/).
+This repository hosts a Linea tracing implementation for 
+[Hyperledger Besu](https://github.com/hyperledger/besu) based on an 
+[existing implementation in Go](https://github.com/Consensys/zk-evm/).
```
Comment (m4sterbunny) [inline README.md:8]:
```suggestion
Tracing refers to the process of extracting data from the execution of an EVM client in order to construct large matrices known as execution traces. Execution traces are subject to the constraint system specified in the [linea-specification](https://github.com/Consensys/linea-specification) repo and implemented in the [linea-constraints](https://github.com/Consensys/linea-constraints) repo.

```

File: README.md [hunk 7]
```diff
@@ -1,182 +1,68 @@
...
+Tracing refers to the process of extracting data from the execution of an EVM client in order to construct large matrices known as execution traces. Execution traces are subject to the constraint system specified in the [linea-specification](https://github.com/Consensys/linea-specification) repo and implemented in the [linea-constraints](https://github.com/Consensys/linea-constraints) repo.
```
Comment (m4sterbunny) [inline README.md:7]:
```suggestion

Tracing refers to the process of extracting data from the execution of an EVM client in order to construct large matrices known as execution traces. Execution traces are subject to the constraint system specified in the [linea-specification](https://github.com/Consensys/linea-specification) repo and implemented in the [linea-constraints](https://github.com/Consensys/linea-constraints) repo.
```

File: README.md [hunk 8]
```diff
@@ -1,182 +1,69 @@
...
-  ![idea_enable_annotation_processing_setting.png](images/idea_enable_annotation_processing_setting.png)
-
-______________________________________________________________________
-
-NOTE
-
-> This setting is required to avoid IDE compilation errors because of the [Lombok](https://projectlombok.org/features/)
-> library used for code generation of boilerplate Java code such as:
+Linea's stack is made up of multiple repositories, these include:
+- [linea-tracer](https://github.com/Consensys/linea-tracer):
```
Comment (m4sterbunny) [inline README.md]:
```suggestion
- This repo, [linea-tracer](https://github.com/Consensys/linea-tracer): Linea-Besu plugin which produces the traces that the constraint system applies and that serve as inputs to the prover
```

File: README.md [hunk 9]
```diff
@@ -1,182 +1,69 @@
-# Linea Arithmetization (zkEVM)
+# Linea tracer (zkEVM)
 
-A Linea tracing implementation for [Hyperledger Besu](https://github.com/hyperledger/besu) based on
-an [existing implementation in Go](https://github.com/Consensys/zk-evm/).
+This repository hosts a Linea tracing implementation for 
+[Hyperledger Besu](https://github.com/hyperledger/besu) based on an 
+[existing implementation in Go](https://github.com/Consensys/zk-evm/).
```
Comment (OlivierBBB) [inline README.md]: This repo (zk-evm / zk-geth) isn't public @m4sterbunny . We recently got some feedback to that effect.

**Review by OlivierBBB — APPROVED** [review]

File: README.md (hunk 9 above)
Comment (m4sterbunny) [inline README.md]:
```suggestion
 implementation in Go.
```

Comment (m4sterbunny) [reply README.md]: OK link removed

**Pull request merged**